virtio-devices/src/vhost_user/blk.rs

```diff
@@ -261,10 +380,20 @@ impl VirtioDevice for Blk {
                 pause_evt,
                 vu_interrupt_list: interrupt_list_sub,
                 slave_req_handler: None,
+                sockfd: Some(self.vhost_user_blk.as_mut().unwrap().as_raw_fd()),
+                disconnected: Some(self.disconnected.clone()),
+                activate_evt: Some(
+                    self.activate_evt
+                        .try_clone()
+                        .map_err(ActivateError::CloneEventfd)
+                        .unwrap(),
+                ),
             });
 
             let paused = self.common.paused.clone();
             let paused_sync = self.common.paused_sync.clone();
+            let mut epoll_threads = Vec::new();
```

Comment:
Why is this here?

You essentially moved the previous `let mut epoll_threads = ...` here.

Reply:
I removed this line and keep the `mut` mentioned in your previous comment.